### PR TITLE
Pull Windows Defender Logs

### PR DESCRIPTION
**Pull request by FredericIV**
SuperCAT currently pulls McAfee logs from computers, but fails if they aren't found. Instead, check and pull both McAfee locations and Windows Defender event logs.

### CODE REVIEW

**Comment by lordneeko** [comment]
@FredericIV  Please re-base all of these PRs to the SuperCAT-main branch so I can see the changes to each one

**Review by lordneeko — COMMENTED** [review]
I do not see changes that appear relevant to pulling Windows Defender logs.

File: Tool/Modules/Antivirus.psm1 [hunk 1]
```diff
@@ -35,16 +35,19 @@ function Import-AntivirusLog {
...
+    if (Test-Path -PathType Leaf -Path "C:\ProgramData\McAfee\DestkopProtection\OnDemandScanLog.txt"){
+        Copy-Item -Path "C:\ProgramData\McAfee\DestkopProtection\OnDemandScanLog.txt" -Destination "$LogPrefix`_OnDemandScanLog.txt"
     }
-    else{
-        Copy-Item -Path "C:\ProgramData\McAfee\Endpoint Security\Logs\" -Destination "$Directory" -Recurse
+    if (Test-Path -PathType Leaf -Path "C:\ProgramData\McAfee\Endpoint Security\Logs\*"){
+        Copy-Item -Path "C:\ProgramData\McAfee\Endpoint Security\Logs\" -Destination "$LogPrefix`_EndpointSecurity\" -Recurse
+    }
+    if ($(wevtutil.exe el).Contains("Microsoft-Windows-Windows Defender/Operational")){
+        wevtutil.exe epl "Microsoft-Windows-Windows Defender/Operational" "$LogPrefix`_WindowsDefender.evtx"
```
Comment (FredericIV) [inline Tool/Modules/Antivirus.psm1]: Windows Defender logs are stored in the EventLog format.

**Review by lordneeko — APPROVED** [review]
Alright I'm seeing it now. Thanks

**Pull request merged**